### ISSUE TITLE
Letzter Commit vor 3 Jahren

### ISSUE THREAD

**Issue by UgnilJoZ** [bundestag/gesetze#55]
Finde ich sehr schade, dass das Repo nicht mehr verwaltet wird…

Obermenge von #54

**Comment by hhirsch** [comment]
Wenn es da im Netz noch Quellen gibt die aktuell gehalten werden, dann können wir ein Skript schreiben um sie dort Runterzuladen evtl. in md umwandeln. @UgnilJoZ Hast du eine idee?

**Comment by UgnilJoZ** [comment]
@hhirsch Theoretisch gibt es ja noch das Repo https://github.com/bundestag/gesetze-tools, das alles mitbringen sollte, um die Gesetze halbautomatisch einzupflegen. Auch dieses wurde vor gut zwei Jahren das letzte Mal geändert, müsste aber noch gehen. Ich werd das mal ausprobieren.

**Comment by stefanw** [comment]
Genau, grundsätzlich sollte alles noch gehen, aber vielleicht eben auch nicht. Gerne auch pull requests an das Repo. Ich gebe auch gerne jedem, der einen vernünftigen Pull Request zustande bringt, Schreibrechte geben.

**Comment by laulens12** [comment]
Kümmert sich denn aktuell jemand darum? Das Gespräch ist ja auch schon eine Weile her... Ich habe leider nicht die Python-Kenntnisse.

**Comment by darkdragon-001** [comment]
Kennt jemand aktuellere Quellen als [gesetzte-im-internet.de](https://www.gesetze-im-internet.de/)?
Dort scheint es zum Beispiel keine aktuelle Version des _Bundesdatenschutzgesetz (BDSG)_ in der Fassung von 2018 zu geben...

**Comment by stefanw** [comment]
@darkdragon-001 gesetze-im-internet.de ist das offizielle Portal für Gesetze. Das Bundesgesetzblatt stellt die Änderungsgesetze zeitnaher bei sich zur Verfügung, die Änderungen müssen dann aber erst (offenbar von Hand) in den Gesetzestext übertragen werden.

**Comment by mdschweda** [comment]
Wie bedauerlich, dass dieses Projekt aufgegeben wurde

**Comment by keilw** [comment]
Schade, wenn es nicht mehr gewartet wird, sollte der Eigentümer das Repository konsequenterweise auf **archived** setzen.

**Comment by ISSTJ** [comment]
Es gibt wieder eine Seite, welche alle aktuellen Gesetztes-Veröffentlichungen online stellt. Die Frage ist, nur, ob diese Plattform nicht vom Dumont Verlag in Grund und Boden geklagt wird.
https://offenegesetze.de/

**Comment by tom-richter** [comment]
[wolterskluwer-online](https://research.wolterskluwer-online.de/) bietet Verlinkungen zu Paragrafen und einen praktischen Versionsvergleich. Zumindest diese beiden Features scheinen kostenlos zu sein.

**Comment by rugk** [comment]
As indicated in https://github.com/bundestag/gesetze/pull/58#issuecomment-1033097288 why not run this update automatically via GitHub actions?

**Comment by mk-pmb** [comment]
I can help with GitHub Actions workflow setup if we decide we want that.

**Comment by ulfgebhardt** [comment]
> I can help with GitHub Actions workflow setup if we decide we want that.

We want that!

Please reference https://github.com/bundestag/gesetze-tools/issues/16
Consider to implement the workflow on the https://github.com/bundestag/gesetze-tools/ repo. ❤️
